client/go/outline/electron/main.go

```diff
@@ -88,6 +98,7 @@ func main() {
 	args.logLevel = flag.String("logLevel", "info", "Logging level: debug|info|warn|error|none")
 	args.dnsFallback = flag.Bool("dnsFallback", false, "Enable DNS fallback over TCP (overrides the UDP handler).")
 	args.checkConnectivity = flag.Bool("checkConnectivity", false, "Check the proxy TCP and UDP connectivity and exit.")
+	args.fetchConfig = flag.String("fetchConfig", "", "Fetch the dynamic key from the given URL and exit.")
```

Comment:
Please organize the flags in groups with comments so we know what flags apply to what operations.

Reply:
Done